.coveragerc

```diff
@@ -1,5 +1,36 @@
 [run]
-omit=*curation_env*
-     *unit_test*
-     *data_steward/lib/*
-     *google-cloud-sdk*
+
```

Comment:
Is this file still needed? Don't the unit/integration versions subsume this?

Reply:
It is still needed.  It is currently the default coverage configuration file.  It is listed as the default in `tests/runner.py`.  This module makes sure a file with the given name exists and if it does not exist, defaults to `curation/.coveragerc`.